src/components/GameCard.tsx

```diff
@@ -30,39 +32,53 @@ export const GameCard: React.FC<GameCardProps> = ({
   }, [game.sequelFamily]);
   const upcoming = useMemo(() => {
     if (game.releaseDate) {
-      return game.releaseDate === "TBA"
-        ? true
-        : new Date(game.releaseDate) > new Date();
-    } else {
-      return false;
+      return (
+        game.releaseDate === "TBA" || new Date(game.releaseDate) > new Date()
+      );
+    }
+
+    if (game.year) {
+      return new Date(game.year) > new Date();
     }
-  }, [game.releaseDate]);
+
+    return false;
+  }, [game.releaseDate, game.year]);
 
   return (
-    <Card className="dark:bg-zinc-900">
-      <CardHeader className="pb-2">
-        {upcoming && (
+    <Card className="dark:bg-zinc-900 h-full flex flex-col">
+      <CardHeader className="pb-2 flex-none">
+        {upcoming ? (
           <div>
             <Badge variant="destructive" className="text-base">
               Not Released
             </Badge>
           </div>
+        ) : (
+          <div>
+            <Badge variant="released" className="text-base">
+              Released
+            </Badge>
+          </div>
         )}
-        {game.releaseDate && (
-          <p className="text-sm mt-1">
-            <span className="font-light">Release Date:</span>{" "}
-            <span className="font-normal">{game.releaseDate}</span>
-          </p>
-        )}
+        <p className="text-sm mt-1">
+          <span className="font-light">Release Date:</span>{" "}
+          <span className="font-normal">
+            {game.releaseDate || "Not available"}
+          </span>
+        </p>
         <CardTitle className="text-xl">{game.title}</CardTitle>
         <p className="text-base">{game.description}</p>
       </CardHeader>
-      <CardContent>
+      <CardContent className="flex-1 flex flex-col justify-between">
         <div>
-          {!!game.year && (
+          {game.year ? (
             <p className="text-sm mb-1 text-gray-500 dark:text-gray-400">
               Year: {game.year}
             </p>
+          ) : (
+            <p className="text-sm mb-1 text-gray-500 dark:text-gray-400">
+              Year: TBA
+            </p>
```

Comment:
Year should be hidden if not defined to avoid repeating TBA

<img width="223" alt="Screenshot 2024-12-19 at 09 39 40" src="https://github.com/user-attachments/assets/ae6043b4-1f78-4b58-82fe-72afead699e3" />

Reply:
Fixed!